fnet/data/aicstiffdataset.py

```diff
@@ -0,0 +1,67 @@
+import torch.utils.data
```

Comment:
@counkomol What would be preferred method of comparing this to `TiffDataset` because you thought these files were redundant.

Reply:
I believe the examples can be redone using TiffDataset.